views/partials/graphmenu.hbs

```diff
@@ -1,15 +1,11 @@
-<section class="ui raised segment">
     <div class="ui inverted blue vertical pointing menu">
-        <a id="burndown"   class= "item" href="/graphs">Burndown</a>
+        <a id="burndown"   class= "item" href="/burndown">Burndown</a>
```

Comment:
stub was in took it out to clean up code.  Future changes not part of this jira.